### ISSUE TITLE
[Insert Modal Field, customNameField, InsertButton, ...etc] How to return defaults from hook functions?

### ISSUE THREAD

**Issue by wzup** [AllenFang/react-bootstrap-table#1435]
How to return defaults from hook functions that change plugin's behavior? Look at these pseudo code examples made from your real examples:

1. [Insert Modal Field](http://allenfang.github.io/react-bootstrap-table/custom.html#insertmodalfield)
```
customNameField = (column, attr, editorClass, ignoreEditable) => {
  if('I need default')
     return [react-bootstrap-table].giveMeDefaultElementHere();
  else
  return (
    <select className={ `${editorClass}` } { ...attr }>
      {
        fruits.map(name => (<option key={ name } value={ name }>{ name }</option>))
      }
    </select>
  );
}
``` 

2. [trClassName](http://allenfang.github.io/react-bootstrap-table/custom.html#insertmodalfield)
```
trClassFormat(row, rowIndex) {
  if('I need default')
     return row.giveMePluginDefaultValue()
  else
  // row is the current row data
  return rowIndex % 2 === 0 ? "tr-odd" : "tr-even";  // return class name.
}
```

**Comment by AllenFang** [comment]
@wzup, sorry for lately reply, it's busy for me :p

For first case, it's a good point, I'll support it in future.

But for second case, `react-bootstrap-table` doesn't have any default class on `tr`, so return `null` or empty string will be fine.

Thanks

**Comment by AllenFang** [comment]
Hello @wzup, for first case, it was fixed on `v3.4.4`, you can return `false` then the default editor will be applied, thanks
